plugins/lighthouse/src/components/Intro/index.tsx

```diff
@@ -0,0 +1,173 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+import { useLocalStorage } from 'react-use';
+import Markdown from 'react-markdown';
+import { ContentHeader, InfoCard } from '@backstage/core';
+import { makeStyles, Button, Grid, Tabs, Tab } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+import LighthouseSupportButton from '../SupportButton';
+
+export const LIGHTHOUSE_INTRO_LOCAL_STORAGE =
+  '@backstage/lighthouse-plugin/intro-dismissed';
+
+const USE_CASES = `
+Google's [Lighthouse](https://developers.google.com/web/tools/lighthouse) auditing tool for websites
+is a great open-source resource forbenchmarking and improving the accessibility, performance, SEO, and best practices of your site.
+At Spotify, we keep track of Lighthouse audit scores over time to look at trends and overall areas for investment.
+
+This plugin allows you to generate on-demand Lighthouse audits for websites, and to track the trends for the
+top-level categories of Lighthouse at a glance.
+
+In the future, we hope to add support for scheduling audits (which we do internally), as well as allowing
+custom runs of Lighthouse to be ingested (for auditing sites that require authentication or some session state).
+`;
+
+const SETUP = `
+To get started, you will need a running instance of [lighthouse-audit-service](https://github.com/spotify/lighthouse-audit-service).
+_It's likely you will need to enable CORS when running lighthouse-audit-service. Initialize the app
+with the environment variable \`LAS_CORS\` set to \`true\`._
+
+When you have an instance running that Backstage can hook into, make sure to export the plugin in
+your app's [\`plugins.ts\`](https://github.com/spotify/backstage/blob/master/packages/app/src/plugins.ts)
+to enable the plugin:
+
+\`\`\`js
+export { default as LighthousePlugin } from '@backstage/plugin-lighthouse';
+\`\`\`
+
+Then, you need to use the \`lighthouseApiRef\` exported from the plugin to initialize the Rest API in
+your [\`apis.ts\`](https://github.com/spotify/backstage/blob/master/packages/app/src/apis.ts).
+
+\`\`\`js
+import { ApiHolder, ApiRegistry } from '@backstage/core';
+import {
+  lighthouseApiRef,
+  LighthouseRestApi,
+} from '@backstage/lighthouse-audits';
+
+const builder = ApiRegistry.builder();
+
+export const lighthouseApi =
+  new LighthouseRestApi(/* your service url here! */);
+builder.add(lighthouseApiRef, lighthouseApi);
+
+export default builder.build() as ApiHolder;
+\`\`\`
+`;
```

Comment:
still feel it's a bit unfortunate to have to use markdown :) is it for the code samples?

Reply:
so! my *actual* vision for this, based on @katz95's designs, is to pull this data directly from the markdown file for the README.md. It's currently an exact copy of what's in the README, but the designs want it to be reflected to the user when first visiting the plugin. So, I created https://github.com/spotify/backstage/issues/515, as it would be really helpful to have a consistent, easy way to do this.

Reply:
Coolers! Not fully sure the same info will typically go in a README (such as installation instructions...) as the about text i kinda envision. But that's ok